packages/tgpu-cli/tgpu-cli.mjs

```diff
@@ -11,18 +14,24 @@ const args = arg({
   '--help': Boolean,
   '--input': String,
   '--output': String,
+  '--commonjs': Boolean,
+  '--js': Boolean,
 
   '-v': '--version',
   '-h': '--help',
   '-i': '--input',
   '-o': '--output',
+  '-c': '--commonjs',
```

Comment:
Imo, we should not give a shortcut for `--commonjs`, hard to interpret `c` as `commonjs` when somebody first sees it.

```suggestion
```